extensions/azurePublishNew/src/types/index.ts

```diff
@@ -1,7 +1,62 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+// https://docs.microsoft.com/en-us/azure/cognitive-services/luis/luis-reference-regions
```

Comment:
this file can semantically be broken into smaller files with a more representative name than index.ts.

there are also some constants in addition to types.